src/routes/docs/tutorials/apple/step-9/+page.markdoc

```diff
@@ -8,3 +8,15 @@ step: 10
 # Test your project {% #test-project %}
 At the top level menu click on **Product** then **Scheme** and click on **Choose scheme**, you will notice that the active scheme is selected if not click on it to mark it selected then next to it select any iOS Simulator you wish to run the app on then click on play button.
 
+# Delete Appwrite Project in Console {% #delete-appwrite-project-in-console %}
```

Comment:
I love that we are showing the user how to remove their project from the console. This is a huge plus in terms of keeping their account secure 😍